app/src/main/java/com/mapbox/maps/testapp/examples/linesandpolygons/DrawGeoJsonLineActivity.kt

```diff
@@ -19,18 +21,26 @@ class DrawGeoJsonLineActivity : AppCompatActivity() {
   public override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.activity_dds_draw_geojson_line)
+    mapView.getMapboxMap().setCamera(
+      CameraOptions.Builder().center(
+        Point.fromLngLat(
+          LATITUDE,
+          LONGITUDE
+        )
+      ).zoom(CAMERA_ZOOM).build()
```

Comment:
Perhaps just `ZOOM` to align with `LATITUDE` and `LONGITUDE`?

Reply:
or even use approach of setting camera as a whole obj from `DrawPolygonActivity.kt`

Reply:
Yeah, here I just wanted to see what are different ways to get camera options.